CodeComplianceTest_Engine/Convert/ToBHoM.cs

```diff
@@ -31,7 +31,7 @@ namespace BH.Engine.Test.CodeCompliance
 {
     public static partial class Convert
     {
-        public static Span ToBHoM(this Microsoft.CodeAnalysis.Text.TextSpan span)
+        public static Span ToSpan(this Microsoft.CodeAnalysis.Text.TextSpan span)
```

Comment:
This needs moving into ToSpan.cs (it already exists) and this file can then be deleted.

Reply:
Done